app-ios/Sources/TimetableFeature/TimetableView.swift

```diff
@@ -8,6 +8,7 @@ import Theme
 public struct TimetableState: Equatable {
     public var dayToTimetable: [DroidKaigi2022Day: Timetable]
     public var selectedDay: DroidKaigi2022Day
+    public var selectedItem: TimetableItemWithFavorite?
```

Comment:
I think this state isn't needed 👀

Reply:
Will remove it 🔥

Reply:
Correct me if I'm wrong, but I guess this is required for opening `SessionView`.

```Swift
// AppView.swift 167-170
case let .timetable(.selectItem(item)):
    state.sessionState = .init(
        timetableItemWithFavorite: item
)
```

Reply:
You were right! Fixed it in https://github.com/DroidKaigi/conference-app-2022/pull/403/commits/becf16344c9fca6c420c0118858ad89980122209